src/Polytopes/Polyhedron/properties.jl

```diff
@@ -420,6 +420,64 @@ function lattice_points(P::Polyhedron)
     end
 end
 
+
+@doc Markdown.doc"""
+    interior_lattice_points(P::Polyhedron)
+
+Return the integer points contained in the interior of the bounded polyhedron
+`P`.
+
+# Examples
+```jldoctest
+julia> c = cube(3)
+A polyhedron in ambient dimension 3
+
+julia> interior_lattice_points(c)
+1-element VectorIterator{PointVector{Polymake.Integer}}:
+ [0, 0, 0]
+```
+"""
+function interior_lattice_points(P::Polyhedron)
+    if pm_polytope(P).BOUNDED
+        lat_pts = pm_polytope(P).INTERIOR_LATTICE_POINTS
+        return VectorIterator{PointVector{Polymake.Integer}}(lat_pts[:, 2:end])
+    else
+        throw(ArgumentError("Polyhedron not bounded"))
+    end
```

Comment:
Normally I would recommend to validate arguments at the start; this leads to less indentation and IMHO makes it easier to follow the code. Of course in trivial cases like this, it's fine either way. So this is a purely optional suggestion
```suggestion
    pm_polytope(P).BOUNDED || throw(ArgumentError("Polyhedron not bounded"))
    lat_pts = pm_polytope(P).INTERIOR_LATTICE_POINTS
    return VectorIterator{PointVector{Polymake.Integer}}(lat_pts[:, 2:end])
```